massa-models/src/mapping_grpc.rs

```diff
@@ -280,11 +280,20 @@ impl From<EventExecutionContext> for grpc::ScExecutionEventContext {
                 .map(|a| a.to_string())
                 .collect(),
             origin_operation_id: value.origin_operation_id.map(|id| id.to_string()),
-            status: vec![
-                value.is_error.into(),
-                value.read_only.into(),
-                value.is_final.into(),
-            ],
+            status: {
```

Comment:
Please tell me where I'm wrong :(
Let's say this code is executed twice.
The first time: value is read_only==true, is_error==false, is_final == true
This leads to status1 == [1,1] 
The second time: value is read_only==true, is_error==true, is_final == false
This also leads to status2 == [1,1] 
If that's correct, how can status1 and status2 be differentiated?

Reply:
It's about the enum identifier

Reply:
Yes I was I a mind loop and i understood right after clicking "Comment" :)